src/components/PostDialog.tsx

```diff
@@ -20,15 +26,35 @@ export const PostDialog: Component<{
   };
 
   return (
-    <dialog ref={dialogRef} onCancel={cancelHandler}>
-      <div>
-        <Logo
-          src={props.frontmatter.logo.src}
-          alt={props.frontmatter.logoAlt}
-        ></Logo>
-        <h2>{props.frontmatter.title}</h2>
-        <h3>{props.frontmatter.company}</h3>
+    <dialog ref={dialogRef} onCancel={cancelHandler} onClick={cancelHandler}>
+      <div
+        class={styles.container}
+        onClick={(e) => {
+          e.stopPropagation();
+        }}
+      >
+        <span class={styles.header}>
+          <Logo
+            src={props.frontmatter.logo.src}
+            alt={props.frontmatter.logoAlt}
+          ></Logo>
+          <div class={styles["title-container"]}>
+            <h3>{props.frontmatter.title}</h3>
+            <span class={styles.subtitle}>
+              <h4>{props.frontmatter.company}</h4>
+              <h4>
+                {dateString(
+                  props.frontmatter.startDate,
+                  props.frontmatter.endDate
+                )}
+              </h4>
+            </span>
+          </div>
+        </span>
         {props.children}
+        <a href="/">
+          <h4>Back</h4>
```

Comment:
This should probably be a button or something, but leaving it at h4 for now because I plan to refactor this component later.